src/index.js

```diff
@@ -139,8 +149,21 @@ async function init(userOptions: any, outputCallback: (output: { text: string, d
 	img.src = options.headerLogoUrl;
 
 	/* Check whether we have a custom color scheme defined and whether we don't use IE */
-	if (options.colorScheme && BrowserDetect.browser !== "MSIE") {
+	if (options.colorScheme && BrowserDetect.browser !== "MSIE" && BrowserDetect.browser !== "Microsoft Edge") {
 		document.documentElement && document.documentElement.style.setProperty("--color", options.colorScheme);
+	} else if (options.colorScheme) {
+		const colorCollection = document.querySelectorAll(".button_template_container, .button, .list_template_element_button,	.cognigy-persistent-menu-item, .quick_reply");
+		changeColor(colorCollection, "color", options.colorScheme);
+
+		const bgColorCollection = document.querySelectorAll(".button_template_text,	.cognigy-chat-state-closed,	.cognigy-chat-state-open, .cognigy-chat-header-container, .cognigy-chat-header-container__open, .cognigy-chat-bot-message");
+		changeColor(bgColorCollection, "background-color", options.colorScheme);
+
+		function changeColor(coll, key, value){
+			for(var i=0, len=coll.length; i<len; i++)
+			{
+				coll[i].style[key] = value;
+			}
+		}
```

Comment:
No logos for most messages

![image](https://user-images.githubusercontent.com/20241539/43000840-563e372c-8c23-11e8-89b0-2fd1f5e1cf38.png)